mpa/cls/__init__.py

```diff
@@ -37,7 +37,8 @@
 import mpa.modules.models.heads.custom_hierarchical_linear_cls_head
 import mpa.modules.models.heads.custom_hierarchical_non_linear_cls_head
 
-import mpa.modules.models.heads.semisl_cls_head
+import mpa.modules.models.heads.semi_linear_cls_head
+import mpa.modules.models.heads.semi_non_linear_cls_head
```

Comment:
I don't know why it has to be separated into two. Are the results different depending on the head? Or do not support it? As far as I know, `semisl_cls_head` can support all models.

Reply:
Yes, `semisl_cls_head` can support all models but if then, for mobilenetv3, there is architecture difference (we can't load model weights from supervised/incremental). To prevent that, I've separated into `semisl_linear_cls_head` and `semisl_non_linear_cls_head`.

Reply:
Does the head have a big impact on the load model? Also, will this change the results? For reference, all of the previous Semi-SL experiments were the results of one head.

Reply:
Well, that's right. If that's the case, please tie those two heads together into one file. semi_sl_head.py

Reply:
Hmm, but other heads are separated, and I followed the convention. Do you think existing files also should be merged in one file?

Reply:
Don't worry about the other files for now.

Reply:
Okay, I'll reflect your comments. Thank you.

Reply:
Can I postpone merging semi-linear head and semi-nonlinear head? I felt we need a design discussion about this.